packer.py

```diff
@@ -48,24 +52,26 @@ def create_bins(bins: dict) -> list:
             bins[bin]['max_weight']))
     return bins_list
 
+
 def refresh_items(unfitted_items, packer):
     for item in unfitted_items:
         packer.add_item(item)
 
+
 def refresh_bin_types(bin_types, packer):
     for bin in bin_types:
         packer.add_bin(bin)
 
-def execute_packing(items_to_fit: list, bin_types: list, visualize=True, export_img=False, textualize=True) -> list:
+
+def execute_packing(items_to_fit: list, bin_types: list, visualize=False, export_img=False, textualize=False) -> list:
     fitted_items = []   # list of solutions
     tree = Tree("Packing list:", highlight=True, hide_root=True)
     while items_to_fit:
-        packing_efficacy = 0
         packer = Packer()
         refresh_bin_types(bin_types, packer)
         refresh_items(items_to_fit, packer)
         packer.pack(bigger_first=True)
-        best_bin = max(packer.bins, key=lambda b: b.efficacy)    # select the best packed bin
+        best_bin = deepcopy(max(packer.bins, key=lambda b: b.efficacy)) # select the best packed bin and copy it
```

Comment:
Maybe it would be good to create `best_bin` getter in `Packer` class with such code  ?

Reply:
You mean to make it a separate method i.ex. `get_best_packed_bin()` to increase readability? Or something more sophisticated?